documentation/staging/content/userguide/platforms/environments.md

```diff
@@ -103,11 +155,18 @@ Oracle Exalogic Elastic Cloud systems can be migrated to this infrastructure wit
 changes, enabling you to preserve your application investment as you adopt modern cloud
 native infrastructure.
 
-#### Microsoft Azure Kubernetes Service (AKS)
+#### Microsoft Azure
+
+There are three different approaches for deploying the operator to Microsoft Azure:
+- Microsoft Azure Platform
+- Microsoft Azure Kubernetes Service (AKS)
+- Oracle WebLogic Server on the Azure Kubernetes Service Marketplace (WLS on AKS Marketplace)
+
```

Comment:
A more accurate rendering is

Oracle WebLogic Server on AKS from the Azure Marketplace (WLS on AKS Marketplace)

there is no term "Azure Kubernetes Service Marketplace".  There is "Azure Kubernetes Service" and "Azure Marketplace".

Reply:
fixed. Please resolve if it looks good to you now

Reply:
Resolved.